docs/rfcs/yyyymmdd-rfc-template.md

```diff
@@ -0,0 +1,64 @@
+# RFC: [Title]
+
+* Author(s): [Authors]
+* Date: [yyyy-mm-dd]
+* Relevant Issue: [Issue URL]
+---
+
+# TL;DR
+
+Required.
+
+TL;DR stands for *too long; didn't read*, so make sure to put only the most essential information here and keep it short.
+
+# Background
+
+Highly recommended.
+
+Your audiences understand your proposal better if they can understand the problen domain first.
+
+# Goals
+
+Required.
+
+* Be very specific about the goals this RFC is aiming for: Given the problem definition, which aspects will be discussed/resolved in this RFC?
+* You should almost always write this section as a bullet list.
```

Comment:
Debatable.

Reply:
No strong opinion on this either. Maybe we can keep this sentence as-is and see how things work.